rosbag2_storage/src/rosbag2_storage/metadata_io.cpp

```diff
@@ -199,6 +199,7 @@ struct convert<rosbag2_storage::BagMetadata>
     node["compression_mode"] = metadata.compression_mode;
     node["relative_file_paths"] = metadata.relative_file_paths;
     node["files"] = metadata.files;
+    node["custom"] = metadata.custom_data;
```

Comment:
Suggestion to rename `custom` field to the `custom_data`.

Reply:
ok changed